.github/workflows/snyk_push.yml

```diff
@@ -14,11 +14,9 @@ jobs:
         persist-credentials: false
     - name: Run Clojure Snyk Scan
       id: scan
-      uses: puppetlabs/security-snyk-clojure-action@v1
+      uses: puppetlabs/security-snyk-clojure-action@v2
       with:
         snykToken: ${{ secrets.SNYK_PE_TOKEN }}
```

Comment:
There is no `SNYK_PE_TOKEN` secret on the this repo, only `SNYK_GENERAL_KEY`. Maybe that is what we want?